src/CoamClient.js

```diff
@@ -405,8 +405,9 @@ class CoamClient {
         return this.__exec(data);
     }
 
-    getUserPermissionsForResourceType(principal, resourceType) {
-        let url = this.__buildUrl(`/auth/access-management/v1/principals/{{principal}}/permissions/{{resourceType}}`, {
+    getUserPermissionsForResourceType(principal, resourceType, include = null, permissionFilters = null) {
+        let urlString = `/auth/access-management/v1/principals/{{principal}}/permissions/{{resourceType}}`.concat(include ? `?include={{include}}`: '').concat(permissionFilters ? `&permissionFilter={{permissionFilters}}`:'');
```

Comment:
Excellent. Now let's enhance the unit test to also add a test for those properties. There's already a test for this method, and we could use a few more test cases.